src/transformers/models/grounding_dino/processing_grounding_dino.py

```diff
@@ -56,6 +64,26 @@ def get_phrases_from_posmap(posmaps, input_ids):
     return token_ids
 
 
+class GroundingDinoProcessorKwargs(ProcessingKwargs, total=False):
+    _defaults = {
+        "text_kwargs": {
+            "add_special_tokens": True,
+            "padding": False,
+            "stride": 0,
+            "return_overflowing_tokens": False,
+            "return_special_tokens_mask": False,
+            "return_offsets_mapping": False,
+            "return_token_type_ids": True,
+            "return_length": False,
+            "verbose": True,
+        },
+        "images_kwargs": {
+            "do_convert_annotations": True,
+            "do_resize": True,
+        },
+    }
```

Comment:
Hmm, from what i understood in the last message, we don't add defaults here unless they were in processor's call before changes. @molbap to have a look

Reply:
@zucchini-nlp I am also confused that @molbap https://github.com/huggingface/transformers/blob/2c8180acd9d7fc9b91f56425e281ac7237ce3305/src/transformers/models/bridgetower/processing_bridgetower.py#L43
According to this file we can use some files of images_kwargs perhaps?
 ```
            "do_normalize": True,
            "do_center_crop": True,
```

Reply:
hey @SangbumChoi @zucchini-nlp ! this is made to hold defaults that are already existing in the processor __call__. In this case groundingdino was *not passing any kwarg to the image processor*. So we need to pass image processing args to it, and not add new defaults: in this case, the key `images_kwargs` can be empty.

Reply:
Now I understood thanks for the clarification!